src/util/CMakeLists.txt

```diff
@@ -5,5 +5,6 @@ target_sources(${ILANG_LIB_NAME} PRIVATE
   ${CMAKE_CURRENT_SOURCE_DIR}/log.cc
   ${CMAKE_CURRENT_SOURCE_DIR}/str_util.cc
   ${CMAKE_CURRENT_SOURCE_DIR}/fs.cc
+  ${CMAKE_CURRENT_SOURCE_DIR}/validate_model.cc
```

Comment:
`util` is for ILA model independent methods, e.g., string helpers, file system helpers. This should not be in the util category.